### PR TITLE
Update Coding-Help.md

### PR DESCRIPTION
**Pull request by noncentz**
__Coding-Help.md__
Remove `<br />` tags

### CODE REVIEW

**Comment by alayek** [comment]
# LGTM :+1: 

@theflametrooper please take a look.

File: Coding-Help.md [hunk 1]
```diff
@@ -5,10 +5,10 @@ I started this list while doing Algorithms, noticing a few things that were comm
 
 2. Comment your code! Can't stress that one enough, if you get lost or take a break then come back you need to know what your code is trying to do. It will also reinforce what the code means to you, and help other people understand what your code does if you get stuck and ask for help. If you've got a any other tips for campers I'd love to see them!
 
-3. Use the chrome javascript debugger console in conjunction with something like netbeans IDE or sublime text to write your code. Both are free to download. <br />
-The free code camp window is great but you can get a lot more feedback on your code's behaviour by using chrome's JS debugger and a bunch of **console.log(myVariableGoesHere);** <br />
-https://developer.chrome.com/devtools/docs/console <br />
-http://www.sublimetext.com/ <br />
+3. Use the chrome javascript debugger console in conjunction with something like netbeans IDE or sublime text to write your code. Both are free to download.
```
Comment (theflametrooper) [inline Coding-Help.md]: ~~something like netbeans IDE~~ something like the netbeans IDE

File: Coding-Help.md [hunk 2]
```diff
@@ -5,10 +5,10 @@ I started this list while doing Algorithms, noticing a few things that were comm
 
 2. Comment your code! Can't stress that one enough, if you get lost or take a break then come back you need to know what your code is trying to do. It will also reinforce what the code means to you, and help other people understand what your code does if you get stuck and ask for help. If you've got a any other tips for campers I'd love to see them!
 
-3. Use the chrome javascript debugger console in conjunction with something like netbeans IDE or sublime text to write your code. Both are free to download. <br />
-The free code camp window is great but you can get a lot more feedback on your code's behaviour by using chrome's JS debugger and a bunch of **console.log(myVariableGoesHere);** <br />
-https://developer.chrome.com/devtools/docs/console <br />
-http://www.sublimetext.com/ <br />
+3. Use the chrome javascript debugger console in conjunction with something like netbeans IDE or sublime text to write your code. Both are free to download.
+The free code camp window is great but you can get a lot more feedback on your code's behaviour by using chrome's JS debugger and a bunch of **console.log(myVariableGoesHere);**
```
Comment (theflametrooper) [inline Coding-Help.md]: ~~window~~ code editor

File: Coding-Help.md [hunk 3]
```diff
@@ -5,10 +5,10 @@ I started this list while doing Algorithms, noticing a few things that were comm
...
 2. Comment your code! Can't stress that one enough, if you get lost or take a break then come back you need to know what your code is trying to do. It will also reinforce what the code means to you, and help other people understand what your code does if you get stuck and ask for help. If you've got a any other tips for campers I'd love to see them!
 
-3. Use the chrome javascript debugger console in conjunction with something like netbeans IDE or sublime text to write your code. Both are free to download. <br />
-The free code camp window is great but you can get a lot more feedback on your code's behaviour by using chrome's JS debugger and a bunch of **console.log(myVariableGoesHere);** <br />
-https://developer.chrome.com/devtools/docs/console <br />
-http://www.sublimetext.com/ <br />
+3. Use the chrome javascript debugger console in conjunction with something like netbeans IDE or sublime text to write your code. Both are free to download.
+The free code camp window is great but you can get a lot more feedback on your code's behaviour by using chrome's JS debugger and a bunch of **console.log(myVariableGoesHere);**
+https://developer.chrome.com/devtools/docs/console
```
Comment (theflametrooper) [inline Coding-Help.md]: Maybe include a "References:" or something like that...

**Comment by theflametrooper** [comment]
### **LGTM :+1:**

File: Coding-Help.md [hunk 4]
```diff
@@ -5,11 +5,13 @@ I started this list while doing Algorithms, noticing a few things that were comm
...
-3. Use the chrome javascript debugger console in conjunction with something like netbeans IDE or sublime text to write your code. Both are free to download. <br />
-The free code camp window is great but you can get a lot more feedback on your code's behaviour by using chrome's JS debugger and a bunch of **console.log(myVariableGoesHere);** <br />
-https://developer.chrome.com/devtools/docs/console <br />
-http://www.sublimetext.com/ <br />
-https://netbeans.org/downloads/
+3. Use the chrome javascript debugger console in conjunction with something like the netbeans IDE or sublime text to write your code. Both are free to download.
+The FreeCodeCamp code editor is great but you can get a lot more feedback on your code's behaviour by using chrome's JS debugger and a bunch of **console.log(myVariableGoesHere);**
+
+  References
+  - https://developer.chrome.com/devtools/docs/console
```
Comment (alayek) [inline Coding-Help.md]: Don't write references with the URL as text.

Could be like this
- [Chrome Console](https://developer.chrome.com/devtools/docs/console)
- [Sublime Text](http://www.sublimetext.com/)
- [Netbeans](https://netbeans.org/downloads/)

**Pull request merged**